src/gatsby-node.js

```diff
@@ -398,9 +398,43 @@ exports.sourceNodes = async (
       }
       if (data['modified']) {
         console.log('we are in modified state');
-        let url = data['modified'][0]['@id'];
+        let urlChild = data['modified'][0]['@id'];
         let urlParent = data['modified'][0]['parent']['@id'];
-        console.log(urlParent);
+        let urlList = [urlChild, urlParent];
+        urlList.forEach(async url => {
+          for await (const node of ploneNodeGenerator(
+            url,
+            token,
+            baseUrl,
+            expansions,
+            backlinks
+          )) {
+            logger.info(
+              `Creating node – ${node.id.replace(baseUrl, '') || '/'}`
+            );
+            createNode(node);
+          }
+        });
+        const childItems = normalizeData(
+          await fetchPlone(
+            `${urlChild}/@search`,
+            token,
+            // Search nodes in path order to ensure parents before their children
+            {
+              ...searchParams,
+              metadata_fields: 'modified',
```

Comment:
Most probably you don’t need any of these extra params after ...searchParams, because you don’t use them.